packages/dashboard-backend/src/routes/api/helpers/__mocks__/getDevWorkspaceClient.ts

```diff
@@ -140,89 +154,90 @@ export const stubAutoProvision = true;
-    } as IServerConfigApi,
-    devworkspaceApi: {
-      create: (_devworkspace, _namespace) =>
-        Promise.resolve({ devWorkspace: stubDevWorkspace, headers: stubHeaders }),
-      delete: (_namespace, _name) => Promise.resolve(undefined),
-      getByName: (_namespace, _name) => Promise.resolve(stubDevWorkspace),
-      listInNamespace: _namespace => Promise.resolve(stubDevWorkspacesList),
-      patch: (_namespace, _name, _patches) =>
-        Promise.resolve({ devWorkspace: stubDevWorkspace, headers: stubHeaders }),
-    } as IDevWorkspaceApi,
-    dockerConfigApi: {
-      read: _namespace => Promise.resolve(stubDockerConfig),
-      update: (_namespace, _dockerCfg) => Promise.resolve(stubDockerConfig),
-    } as IDockerConfigApi,
-    kubeConfigApi: {
-      injectKubeConfig: (_namespace, _devworkspaceId) => Promise.resolve(undefined),
-    } as IKubeConfigApi,
-    devWorkspaceTemplateApi: {
-      create: _template => Promise.resolve(stubDevWorkspaceTemplate),
-      listInNamespace: _namespace => Promise.resolve(stubDevWorkspaceTemplatesList),
-      getByName: (_namespace, _name) => Promise.resolve(stubDevWorkspaceTemplate),
-      patch: (_namespace, _name, _patches) => Promise.resolve(stubDevWorkspaceTemplate),
-      delete: (_namespace, _name) => Promise.resolve(undefined),
-    } as IDevWorkspaceTemplateApi,
-    userProfileApi: {
-      getUserProfile: _namespace => Promise.resolve(stubUserProfile),
-    } as IUserProfileApi,
-    editorsApi: {
-      list: () => Promise.resolve(editorsArray),
-    } as IEditorsApi,
-    eventApi: {
-      listInNamespace: _namespace => Promise.resolve(stubEventsList),
-      watchInNamespace: _namespace => Promise.resolve(),
-      stopWatching: () => undefined,
-    } as IEventApi,
-    podApi: {
-      listInNamespace: _namespace => Promise.resolve(stubPodsList),
-      stopWatching: () => undefined,
-      watchInNamespace: _namespace => Promise.resolve(),
-    } as IPodApi,
-    logsApi: {
-      stopWatching: () => undefined,
-      watchInNamespace: (_namespace, _name) => Promise.resolve(),
-    } as ILogsApi,
-    personalAccessTokenApi: {
-      create: (_namespace, _token) => Promise.resolve({} as api.PersonalAccessToken),
-      delete: (_namespace, _tokenName) => Promise.resolve(),
-      listInNamespace: _namespace => Promise.resolve(stubPersonalAccessTokenList),
-      replace: (_namespace, _token) => Promise.resolve({} as api.PersonalAccessToken),
-    } as IPersonalAccessTokenApi,
-    gitConfigApi: {
-      read: _namespace => Promise.resolve({} as api.IGitConfig),
-      patch: (_namespace, _gitconfig) => Promise.resolve({} as api.IGitConfig),
-    } as IGitConfigApi,
-    sshKeysApi: {
-      add: (_namespace, _sshKey) => Promise.resolve({} as api.SshKey),
-      delete: (_namespace, _name) => Promise.resolve(),
-      list: _namespace => Promise.resolve(stubSshKeysList),
-    } as IShhKeysApi,
-  } as DevWorkspaceClient;
-}
+export const getDevWorkspaceClient = jest.fn(
```

Comment:
This was done in order to provide certain mock implementations for certain tests:
https://github.com/eclipse-che/che-dashboard/blob/64cb29d42acfc59104aac5ffd76ba5d221b56be0/packages/dashboard-backend/src/routes/api/__tests__/editors.spec.ts#L67